consensus/istanbul/core/handler.go

```diff
@@ -156,7 +156,17 @@ func (c *core) handleMsg(payload []byte) error {
 	msg := new(message)
 	if err := msg.FromPayload(payload, c.validateFn); err != nil {
 		if c.backend.NodeType() == common.CONSENSUSNODE {
-			logger.Error("Failed to decode message from payload", "err", err)
+			if err != istanbul.ErrUnauthorizedAddress {
+				logger.Error("Failed to decode message from payload", "err", err)
+			} else {
+				// Decode msg to derive view, so node's view and msg's view can be compared
+				msgView, msgDecodeErr := msg.GetView()
+				if msgDecodeErr != nil {
+					logger.Error("Failed to decode message", "code", msg.Code, "err", msgDecodeErr)
+					return errInvalidMessage
+				}
+				logger.Warn("Signed by an unauthorized address. If syncing, it may or may not be an unauthorized address.", "node view", c.currentView().String(), "msg view", msgView.String())
```

Comment:
The sentence is too ambiguous 
```suggestion
				logger.Warn("Signed by an unauthorized address. It could happen when there is an unsynced validator in the network", "node view", c.currentView().String(), "msg view", msgView.String())
```